hack/preview.sh

```diff
@@ -179,6 +179,7 @@ sed -i.bak "s/rekor-server.enterprise-contract-service.svc/$rekor_server/" $ROOT
 
 [ -n "${RELEASE_IMAGE_REPO}" ] && yq -i e "(.images.[] | select(.name==\"quay.io/redhat-appstudio/release-service\")) |=.newName=\"${RELEASE_IMAGE_REPO}\"" $ROOT/components/release/kustomization.yaml
 [ -n "${RELEASE_IMAGE_TAG}" ] && yq -i e "(.images.[] | select(.name==\"quay.io/redhat-appstudio/release-service\")) |=.newTag=\"${RELEASE_IMAGE_TAG}\"" $ROOT/components/release/kustomization.yaml
+[[ -n "${RELEASE_SERVICE_PR_OWNER}" && "${RELEASE_SERVICE_PR_SHA}" ]] && yq -i e "(.resources[] | select(. ==\"*github.com/redhat-appstudio/release-service*\")) |= \"https://github.com/${RELEASE_SERVICE_PR_OWNER}/release-service/config/default?ref=${RELEASE_SERVICE_PR_SHA}\"" $ROOT/components/release/kustomization.yaml
```

Comment:
@davidmogar good point, thanks. Am I right thinking that `RELEASE_RESOURCES` was only used as a means to manually override this if needed and it was not used in any automation? So we can remove it, because if you need a custom ref, you can use the new env vars going forward?

Reply:
The only automation I know of is this one https://github.com/redhat-appstudio/release-service/blob/main/.tekton/push.yaml and it doesn't use it. AFAIK it's only used in this script, so `RELEASE_RESOURCE` can be removed. We will use the new two variables. Just announce it on the team chat once merged :wink:

Reply:
OK, will do. If I ever get the PR test run to pass ;)

Reply:
Fixed.